spec/home.spec.tsx

```diff
@@ -1,24 +1,53 @@
-import { test, expect } from "vitest"
+import { test, expect, vi } from "vitest"
 import { render, screen } from "@testing-library/react"
 import { NostoProvider, NostoHome, NostoPlacement } from "../src/index"
 import RecommendationComponent from "./renderer"
 import { waitForRecommendations } from "./utils"
+import { jsonMockDataForPage } from "./mocks"
 
 test("Home page render", async () => {
+  const actionMock = {
```

Comment:
can you move lines 9-36 also to mocks?
and already sketch how this mock is parametrized

Reply:
Yes, it's already planned for when I spread the usage. Thanks!